### ISSUE TITLE
Need to act on Debian Bug 1025620 (pkexec dependency)

### ISSUE THREAD

**Issue by sriemer** [scanmem/scanmem#416]
See: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1025620

Looks like I’m still the Debian maintainer for GameConqueror. But my focus shifted. Got a toddler that needs my attention more. So I need support on this one as I do not have the time to do it on my own.

Anyone with Debian packaging experience interested in helping me out to keep GameConqueror in Debian based distros?

@12345ieee, @kartikm ?

**Comment by kartikm** [comment]
I'll take care of this. Thanks for heads up!

**Comment by sriemer** [comment]
Thanks so much, Kartik. :smiley:

**Comment by kartikm** [comment]
Fixed in Debian package 0.17-4

**Comment by 12345ieee** [comment]
Thanks @kartikm .

**Comment by sriemer** [comment]
The scanmem package 0.17-4 has been marked for auto-removal on **2023-03-05** now: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1030481 due to a serious bug report. So reopened this.
@kartikm ?

**Comment by sriemer** [comment]
Looks like `dh_missing` is complaining about missing installation of Python cache files.

**Comment by 12345ieee** [comment]
Yeah, the bug seems on the debian side tbh, about how the debian build deals with pycache files.
I don't think upstream needs to do anything, but I offer any support needed.

**Comment by kartikm** [comment]
@sriemer I've fixed and uploaded the package. Let's see if that builds well in all arches.
